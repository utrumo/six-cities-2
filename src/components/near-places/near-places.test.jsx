import React from 'react';
import renderer from 'react-test-renderer';
import NearPlaces from './near-places.jsx';

it(`NearPlaces correctly renders after restart`, () => {
  const nearPlaces = renderer.create(<NearPlaces />).toJSON();
  expect(nearPlaces).toMatchSnapshot();
});
