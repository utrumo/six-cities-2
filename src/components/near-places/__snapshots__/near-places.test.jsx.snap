// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NearPlaces correctly renders after restart 1`] = `
<div
  className="container"
>
  <section
    className="near-places places"
  >
    <h2
      className="near-places__title"
    >
      Other places in the neighbourhood
    </h2>
    <div
      className="near-places__list places__list"
    >
      <article
        className="near-places__card place-card"
      >
        <div
          className="near-places__image-wrapper place-card__image-wrapper"
        >
          <a
            href="#"
          >
            <img
              alt="Place image"
              className="place-card__image"
              height="200"
              src="/img/room.jpg"
              width="260"
            />
          </a>
        </div>
        <div
          className="place-card__info"
        >
          <div
            className="place-card__price-wrapper"
          >
            <div
              className="place-card__price"
            >
              <b
                className="place-card__price-value"
              >
                €80
              </b>
              <span
                className="place-card__price-text"
              >
                / night
              </span>
            </div>
            <button
              className="place-card__bookmark-button place-card__bookmark-button--active button"
              type="button"
            >
              <svg
                className="place-card__bookmark-icon"
                height="19"
                width="18"
              >
                <use
                  xlinkHref="#icon-bookmark"
                />
              </svg>
              <span
                className="visually-hidden"
              >
                In bookmarks
              </span>
            </button>
          </div>
          <div
            className="place-card__rating rating"
          >
            <div
              className="place-card__stars rating__stars"
            >
              <span
                style={
                  Object {
                    "width": "80%",
                  }
                }
              />
              <span
                className="visually-hidden"
              >
                Rating
              </span>
            </div>
          </div>
          <h2
            className="place-card__name"
          >
            <a
              href="#"
            >
              Wood and stone place
            </a>
          </h2>
          <p
            className="place-card__type"
          >
            Private room
          </p>
        </div>
      </article>
      <article
        className="near-places__card place-card"
      >
        <div
          className="near-places__image-wrapper place-card__image-wrapper"
        >
          <a
            href="#"
          >
            <img
              alt="Place image"
              className="place-card__image"
              height="200"
              src="/img/apartment-02.jpg"
              width="260"
            />
          </a>
        </div>
        <div
          className="place-card__info"
        >
          <div
            className="place-card__price-wrapper"
          >
            <div
              className="place-card__price"
            >
              <b
                className="place-card__price-value"
              >
                €132
              </b>
              <span
                className="place-card__price-text"
              >
                / night
              </span>
            </div>
            <button
              className="place-card__bookmark-button button"
              type="button"
            >
              <svg
                className="place-card__bookmark-icon"
                height="19"
                width="18"
              >
                <use
                  xlinkHref="#icon-bookmark"
                />
              </svg>
              <span
                className="visually-hidden"
              >
                To bookmarks
              </span>
            </button>
          </div>
          <div
            className="place-card__rating rating"
          >
            <div
              className="place-card__stars rating__stars"
            >
              <span
                style={
                  Object {
                    "width": "80%",
                  }
                }
              />
              <span
                className="visually-hidden"
              >
                Rating
              </span>
            </div>
          </div>
          <h2
            className="place-card__name"
          >
            <a
              href="#"
            >
              Canal View Prinsengracht
            </a>
          </h2>
          <p
            className="place-card__type"
          >
            Apartment
          </p>
        </div>
      </article>
      <article
        className="near-places__card place-card"
      >
        <div
          className="near-places__image-wrapper place-card__image-wrapper"
        >
          <a
            href="#"
          >
            <img
              alt="Place image"
              className="place-card__image"
              height="200"
              src="/img/apartment-03.jpg"
              width="260"
            />
          </a>
        </div>
        <div
          className="place-card__info"
        >
          <div
            className="place-card__price-wrapper"
          >
            <div
              className="place-card__price"
            >
              <b
                className="place-card__price-value"
              >
                €180
              </b>
              <span
                className="place-card__price-text"
              >
                / night
              </span>
            </div>
            <button
              className="place-card__bookmark-button button"
              type="button"
            >
              <svg
                className="place-card__bookmark-icon"
                height="19"
                width="18"
              >
                <use
                  xlinkHref="#icon-bookmark"
                />
              </svg>
              <span
                className="visually-hidden"
              >
                To bookmarks
              </span>
            </button>
          </div>
          <div
            className="place-card__rating rating"
          >
            <div
              className="place-card__stars rating__stars"
            >
              <span
                style={
                  Object {
                    "width": "100%",
                  }
                }
              />
              <span
                className="visually-hidden"
              >
                Rating
              </span>
            </div>
          </div>
          <h2
            className="place-card__name"
          >
            <a
              href="#"
            >
              Nice, cozy, warm big bed apartment
            </a>
          </h2>
          <p
            className="place-card__type"
          >
            Apartment
          </p>
        </div>
      </article>
    </div>
  </section>
</div>
`;
